### PR TITLE
Release 7.6.0 version of the che-plugin-registry

### PR DESCRIPTION
**Pull request by ibuziuk**
### What does this PR do?

Release 7.6.0 version of the che-plugin-registry - https://github.com/eclipse/che/issues/15504

- Bumping VERSION file
- Adding '7.6.0' versions for che-theia & machine-exec 
   -  **NOTE** che-theia 7.6.0 is supposed to be build on quay -  https://quay.io/repository/eclipse/che-theia
- Setting 'latest' version for che-theia and machine-exec plugins to '7.6.0' 
- Setting '7.6.0' version for container runtime images of the che remote plugins - `find . -name '*.yaml' -exec sed -i -e 's/:next/:7.6.0/g' {} \;`
   -  **NOTE** not sure if this step is needed after - https://github.com/eclipse/che-plugin-registry/pull/306 

It is strange though that `eclipse/che-remote-plugin-runner-java8` is still on dockerhub

### CODE REVIEW

File: v3/plugins/broadcommfd/cobol-language-support/0.9.1/meta.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ repository: https://github.com/eclipse/che-che4z-lsp-for-cobol
 category: Language
 spec:
   containers:
-    - image: eclipse/che-remote-plugin-runner-java8:next
+    - image: eclipse/che-remote-plugin-runner-java8:7.6.0
```
Comment (benoitf) [inline v3/plugins/broadcommfd/cobol-language-support/0.9.1/meta.yaml]:
```suggestion
    - image: quay.io/eclipse/che-sidecar-java:8-0cfbacb
```

File: v3/plugins/broadcommfd/debugger-for-mainframe/1.0.0/meta.yaml [hunk 2]
```diff
@@ -11,7 +11,7 @@ repository: https://techdocs.broadcom.com/content/broadcom/techdocs/us/en/ca-mai
 category: Debugger
 spec:
   containers:
-    - image: eclipse/che-remote-plugin-runner-java8:next
+    - image: eclipse/che-remote-plugin-runner-java8:7.6.0
```
Comment (benoitf) [inline v3/plugins/broadcommfd/debugger-for-mainframe/1.0.0/meta.yaml]:
```suggestion
    - image: quay.io/eclipse/che-sidecar-java:8-0cfbacb
```

File: v3/plugins/broadcommfd/cobol-language-support/0.9.1/meta.yaml (hunk 1 above)
Comment (benoitf) [reply v3/plugins/broadcommfd/cobol-language-support/0.9.1/meta.yaml]: it should be fixed in master branch as well

File: v3/plugins/broadcommfd/debugger-for-mainframe/1.0.0/meta.yaml (hunk 2 above)
Comment (benoitf) [reply v3/plugins/broadcommfd/debugger-for-mainframe/1.0.0/meta.yaml]: it should be fixed in master branch as well

**Comment by benoitf** [comment]
FYI it seems ECA check is failing

**Comment by ibuziuk** [comment]
FYI it seems ECA check is failing

@benoitf I'm not sure why. I have added `Signed-off-by: Ilya Buziuk <ibuziuk@redhat.com>` to the last commit

File: v3/plugins/broadcommfd/cobol-language-support/0.9.1/meta.yaml (hunk 1 above)
Comment (amisevsk) [reply v3/plugins/broadcommfd/cobol-language-support/0.9.1/meta.yaml]: Should be confirmed with @filipkroupa -- they target specific versions for compatibility AFAIK.

**Comment by amisevsk** [comment]
I don't see an ECA check on this PR, is it disabled?

File: v3/plugins/broadcommfd/cobol-language-support/0.9.1/meta.yaml (hunk 1 above)
Comment (filipkroupa) [reply v3/plugins/broadcommfd/cobol-language-support/0.9.1/meta.yaml]: @amisevsk As per your recommendation we removed the necessity of specific plugins version, so in this case we should be ok. But I don't know what is the difference between the two docker images. We always used `eclipse/che-remote-plugin-runner-java8` for Java 8 runtime. Should we move to the new docker image `quay.io/eclipse/che-sidecar-java`?

**Pull request merged**